resources/multisite_drupal_standard.make

```diff
@@ -937,7 +937,7 @@ libraries[respond][download][url] = "https://raw.githubusercontent.com/scottjehl
 
 projects[bootstrap][type] = theme
 projects[bootstrap][download][type] = get
-projects[bootstrap][download][url] = http://ftp.drupal.org/files/projects/bootstrap-7.x-3.5.zip
+projects[bootstrap][download][url] = https://ftp.drupal.org/files/projects/bootstrap-7.x-3.8.zip
```

Comment:
what's wrong with the standard format?
`projects[bootstrap][type] = theme`
`projects[bootstrap][version] = 3.8`
I've double checked it in a simple site.make and it works like a charm.